run_all_tests.sh

```diff
@@ -95,5 +101,6 @@ echo ""
 echo "Test Results: ${FAILURE} of ${TOTAL} tests failed (${SKIP} skipped) in ${ELAPSED} seconds."
 echo ""
 
+exit $FAILURE
```

Comment:
Let's make this one: 

if  [ "$FAILURE" = 0 ]
then
 exit 0
else
 exit 1
fi

or something of the sort. We don't want the exit status to vary according to the number of failures.